css-basics/css-basics_combined-users/css/styles.css

```diff
@@ -46,23 +46,47 @@ li {
   background-color: white;
 }
 
+ul .user .bio {
+  visibility: hidden;
+}
+ul .user .location {
+  text-align: left;
+  color: darkslategray;
+}
+
+aside .user {
+  max-width: 250px;
+  padding: 12px;
+  border-radius: 4px;
+}
 .name {
   font-size: 18px;
   text-transform: uppercase;
   margin: 0;
 }
 
+aside .name {
```

Comment:
maybe this needs to be `aside .user .name` to specify the .name's in the .user's?

Reply:
I know what you mean. But both ways have the same end result so i'll just leave it like this coz i'm lazy.